package.json

```
@@ -23,6 +28,10 @@
     "format": "prettier --write \"src/**/*.{js,jsx}\"",
     "lint": "eslint \"src/**/*.{js, jsx}\" --quiet"
   },
+  "engines": {
+    "node": "^18.15.0",
+    "npm": "^9.5.0"
```

Here more generic version is preferred as it will block to run the project without the exact version.
```suggestion
    "node": "^18.x",
    "npm": "^9.5.x"
```